DDDEastAnglia/DataAccess/SponsorSorter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.Models;
+
+namespace DDDEastAnglia.DataAccess
+{
+    public interface ISponsorSorter
+    {
+        IEnumerable<Sponsor> Sort(IEnumerable<Sponsor> sponsors);
+    }
+
+    public class SponsorSorter : ISponsorSorter
```

Comment:
It is probably a default one, as I can't see why we would want to sort them in a different order.